tests/testTriggers.go

```diff
@@ -654,6 +656,79 @@ func TriggerRebootNodes(contexts *[]*scheduler.Context, recordChan *chan *EventR
 	})
 }
 
+// TriggerCrashNodes crashes node on which apps are running
+func TriggerCrashNodes(contexts *[]*scheduler.Context, recordChan *chan *EventRecord) {
+	defer ginkgo.GinkgoRecover()
+	event := &EventRecord{
+		Event: Event{
+			ID:   GenerateUUID(),
+			Type: CrashNode,
+		},
+		Start:   time.Now().Format(time.RFC1123),
+		Outcome: []error{},
+	}
+
+	defer func() {
+		event.End = time.Now().Format(time.RFC1123)
+		*recordChan <- event
+	}()
+
+	Step("get all nodes and crash one by one", func() {
+		nodesToCrash := node.GetWorkerNodes()
```

Comment:
the definition of the test says `crashes node on which apps are running` but it crashes all worker nodes

Reply:
Modified the definintion